blocks/property-listing/search.js

```diff
@@ -0,0 +1,48 @@
+import SearchParameters, { SortDirections, SortOptions } from '../../scripts/apis/creg/SearchParameters.js';
+import renderCards from './cards/cards.js';
+
+// function addMoreButton() {
+// TODO: add this logic if there's supposed to be more results;
+// }
```

Comment:
do we need this code do be commented? Have we created ticket to solve this todo?

Reply:
It's not required for property listing, but is part of the property search page's results. I can take it out, but i can't uncomment it or, the Linter will complain.